website/integrations/sources/apple/index.md

```diff
@@ -10,6 +10,10 @@ Allows users to authenticate using their Apple ID.
 An Apple developer account is required for this.
 :::
 
+:::warning
+Apple only allows OAuth with public internet facing URIs, this will not work for intranet
```

Comment:
`Apple mandates the use of a [registered TLD](https://en.wikipedia.org/wiki/List_of_Internet_top-level_domains), as such this source will not work with .local and other non-public TLDs.`

Reply:
That works for me.